zadaci/sa-kolokvijuma/2020/PSI/T34/G2/index.rst

```diff
@@ -0,0 +1,17 @@
+.. index::
+   Datoteke ; Tekstualne
+   Statičke strukture ; Niz
+
+
+Zadatak "Mobilni"
+=================================
```

Comment:
The number of `=` should follow the length of the section header, please shorten it appropriately.

Reply:
And perhaps consider having a more descriptive assignment title?